source/Plugins/SymbolFile/PDB/SymbolFilePDB.cpp

```diff
@@ -957,7 +957,8 @@ VariableSP SymbolFilePDB::ParseVariableForPDBData(
 
   var_sp = std::make_shared<Variable>(
       var_uid, var_name.c_str(), mangled_cstr, type_sp, scope, context_scope,
-      ranges, &decl, location, is_external, is_artificial, is_static_member);
+      ranges, &decl, location, is_external, is_artificial, is_static_member,
```

Comment:
We should make this an enum.

Reply:
Agreed, but not in swift-lldb. I'll do this when I'm upstreaming this patch.